.github/workflows/set-pull-expectations.yml

```diff
@@ -0,0 +1,19 @@
+name: Set Pull Expectations
+on:
+  pull_request:
```

Comment:
The action doesn't seem to support pull_requests? 
https://github.com/actions/github/blob/master/entrypoint.js